akka-actor/src/main/scala/akka/util/ByteString.scala

```diff
@@ -528,7 +528,7 @@ object CompactByteString {
     else {
       val ar = new Array[Byte](bytes.size)
       bytes.copyToArray(ar)
-      CompactByteString(ar)
+      ByteString.ByteString1C(ar)
```

Comment:
Thanks for testing it, @alno 
I was only thinking of using `toArray.clone` in `def apply(bytes: Byte*): CompactByteString` where we know that it is an array. Or do you mean that varargs can be something else than a (wrapped) array? If that is the case I think we should stick to the copy, especially since the performance difference was not big.